.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
-  "single-quote": true,
-  "trailing-comma": "all",
-  "use-tabs": true,
-  "print-width": 120
+  "singleQuote": true,
+  "trailingComma": "all",
+  "useTabs": true,
+  "printWidth": 120
```

Comment:
Maybe make the `printWidth` a bit smaller? Something like 80?

Reply:
Hmm, good idea. I'm not sure where this came from originally, but the prettier docs do specifically recommend against it: https://prettier.io/docs/en/options.html#print-width.

I've updated it (actually just removed this line, since 80 is the default), and I agree it looks better. Done :+1: